starky/src/get_challenges.rs

```diff
@@ -337,7 +703,17 @@ impl<const D: usize> StarkProofWithPublicInputsTarget<D> {
         C::Hasher: AlgebraicHasher<F>,
     {
         challenger.observe_elements(&self.public_inputs);
-        self.proof
-            .get_challenges::<F, C>(builder, challenger, challenges, ignore_trace_cap, config)
+        self.proof.get_challenges::<F, C, S>(
+            builder,
+            stark,
+            &self.public_inputs,
+            challenger,
+            challenges,
+            ctl_vars,
+            degree_bits,
+            self.proof.degree_bits,
```

Comment:
We're calling `get_challenges` which is a method of `self.proof` so `self.proof.degree_bits` does not need to be an argument